src/patchkernel/patch_kernel.cpp

```diff
@@ -523,7 +523,7 @@ void PatchKernel::initialize()
 	initializeHaloSize(haloSize);
 
 	// Mark patch as partioned
-	if (isPartitioned()) {
+	if (getCommunicator() != MPI_COMM_NULL) {
```

Comment:
Function isCommunicatorSet() could be called, @andrea-iob?

But this doesn't mean that the patch is partitioned, only that the communicator is set. 

The meaning of _partitioned_ patch, from related method isPartitioned(), seems to be that the number of processors is greater than one.

What's the meaning of PARTITIONING_CLEAN status? And when is valid PARTITIONING_UNSUPPORTED? 
A single processor communicator (not null) should be related to a PARTITIONING_UNSUPPORTED patch or always PARTITIONING_CLEAN status?

Reply:
PARTITIONING_UNSUPPORTED means that the patch is not able to partition itself, like the VolCartesian patch.

PARTITIONING_CLEAN means that there are no partitioning operation in progress (i.e., we are not in between one of the prepare/alter/finalize methods). 

The correct way to check if the patch is partitioned is isPartitioned(). The partitioning status is exposed, but I've never used it outside bitpit. It may be useful for checking if the patch support partitioning.